.evergreen.yml

```diff
@@ -124,6 +124,23 @@ tasks:
 
         COVERALLS_REPO_TOKEN=${coveralls_repo_token|} poetry run coveralls
 
+- name: evg_api_install
+  commands:
+  - command: shell.exec
+    params:
+      working_dir: src
+      script: |
+        set -o errexit
+        set -o verbose
+
+        export LC_ALL=C.UTF-8
+        export LANG=C.UTF-8
+
+        . venv/bin/activate
+
+        poetry install
+        poetry run evg-api --help
```

Comment:
I believe this is sufficient to test proper installation.

Reply:
would it be safer to assume users would install this using *pip* instead of *poetry*?